src/components/Profile/components/PersonalInfoList/index.js

```diff
@@ -264,10 +264,17 @@ const PersonalInfoList = ({
   const note =
     myProf &&
     (isFacStaff ? (
-      <Typography align="left" className="note">
-        NOTE: To update your data, please contact{' '}
-        <a href="mailto: hr@gordon.edu">Human Resources</a> (x4828).
-      </Typography>
+      <div align="left" className="note">
+        <Typography>NOTE:</Typography>
+        <ul>
+          <li>
```

Comment:
why this change? I think the old one looks good to me. would you mind explain it to me please?

Reply:
I think I fixed it in this newest commit.